### PR TITLE
Fixed Plasm PC2 spec types

### PR DESCRIPTION
**Pull request by akru**
**Description**

Fix `Address`-related types, added missed block types.

### CODE REVIEW

File: packages/apps-config/src/api/spec/plasm-parachain.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    relay_chain_state: 'Vec<Bytes>'
+  },
+  PersistedValidationData: {
+    parent_head: 'Bytes',
+    block_number: 'u32',
+    relay_storage_root: 'Hash',
+    hrmp_mqc_heads: 'Vec<(u32, Hash)>',
+    dmq_mqc_head: 'Hash',
+    max_pov_size: 'u32'
+  }
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/plasm-parachain.ts:21]: These 2 applies to all, some I missed so should be added to https://github.com/polkadot-js/api/blob/master/packages/types/src/interfaces/parachains/definitions.ts and removed here.

(Happy to do it if you are not up to it, it should just be a copy and changing `snake_case` to `camelCase`)

File: packages/apps-config/src/api/spec/plasm-parachain.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  Address: 'MultiAddress',
+  LookupSource: 'MultiAddress',
+  ValidationDataType: {
+    validation_data: 'ValidationData',
+    relay_chain_state: 'Vec<Bytes>'
+  },
+  PersistedValidationData: {
+    parent_head: 'Bytes',
+    block_number: 'u32',
+    relay_storage_root: 'Hash',
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/plasm-parachain.ts:17]: This field in missing in the API.

**Review by jacogr — COMMENTED** [review]
Would just remove the parachain-specific types (the API PR in https://github.com/polkadot-js/api/pull/3050 will be pulled-in as soon as CI is done)

File: packages/apps-config/src/api/spec/plasm-parachain.ts (hunk 2 above)
Comment (akru) [reply packages/apps-config/src/api/spec/plasm-parachain.ts:17]: This field still available in `rococo-v1` Polkadot primitives: https://github.com/paritytech/polkadot/blob/rococo-v1/primitives/src/v1.rs#L366

**Comment by akru** [comment]
@jacogr fixed, thank you!

File: packages/apps-config/src/api/spec/plasm-parachain.ts (hunk 2 above)
Comment (jacogr) [reply packages/apps-config/src/api/spec/plasm-parachain.ts]: Yes, I meant that the API definition didn't have that field.

File: packages/apps-config/src/api/spec/plasm-parachain.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// Copyright 2017-2021 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+export default {
+  Address: 'MultiAddress',
+  LookupSource: 'MultiAddress',
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/plasm-parachain.ts:9]:
```suggestion
  LookupSource: 'MultiAddress'
```

Comment (jacogr) [reply packages/apps-config/src/api/spec/plasm-parachain.ts:9]: Linting will complain about this one.

**Review by jacogr — APPROVED** [review]

**Pull request merged**